zallet/src/components/keystore/db/migrations/initial_setup.rs

```diff
@@ -0,0 +1,45 @@
+use std::collections::HashSet;
+
+use schemerz_rusqlite::RusqliteMigration;
+use uuid::Uuid;
+use zcash_client_sqlite::wallet::init::{migrations::V_0_15_0, WalletMigrationError};
+
+pub(super) const MIGRATION_ID: Uuid = Uuid::from_u128(0xadfd5ac7_927f_4288_9cca_55520c4b45d1);
+
+pub(super) struct Migration;
+
+impl schemerz::Migration<Uuid> for Migration {
+    fn id(&self) -> Uuid {
+        MIGRATION_ID
+    }
+
+    fn dependencies(&self) -> HashSet<Uuid> {
+        V_0_15_0.iter().copied().collect()
```

Comment:
This should be `V_0_16_0`, but we keep forgetting to create these constants during the `zcash_client_sqlite` release process.